main/webapp/modules/core/langs/translation-en.json

```diff
@@ -680,6 +681,18 @@
     "core-views/use-values-as-identifiers2": "Mark cells as reconciled with their values as identifiers",
     "core-views/choose-reconciliation-service": "Choose a reconciliation service",
     "core-views/choose-reconciliation-service-alert": "Please choose a reconciliation service first.",
+    "core-views/column-join": "Join columns",
+    "core-views/column-join-before-column-picker": "Select and order columns to join",
+    "core-views/column-join-before-options": "Select options",
+    "core-views/column-join-field-separator": "Separator between the content of each column :",
+    "core-views/column-join-field-separator-advice": "Enter one or more character, or keep blank to juxtapose the columns.",
```

Comment:
grammar improvement, replace "character" with "characters".
Also I think "juxtapose" is an often confusing term and translations will be more coherent if we use "append" or even just "connect" instead of "juxtapose".

Reply:
ok for connect